### DISCUSSION TITLE
Continuously receiving "Invalid email / username or password" with using Azure Active Directory

### DISCUSSION THREAD

**Discussion by mike35x95x1** [requarks/wiki#3121] [Help / Questions]
I have Azure AD authentication strategy. I followed all the instructions (https://docs.requarks.io/auth/azure).

When I log-in (using my Azure AD credentials) the callback page displays error:

### URL: https://wiki.[ourdomain].com/login/[GUID]/callback
```
Oops, something went wrong...
Invalid email / username or password.
```

Edit:
**Deleting cookies helps** but every new user must delete the cookies manually.

I found this pull request and tried to apply but it looks like it doesn't help:
https://github.com/Requarks/wiki/pull/2567

Versions:
Ubuntu: 18.04
Wikijs: 2.5.170
Node: 15.9.0
PostgreSQL: 13.2

**Comment by AntonAndreevichMoroz** [comment]
Check the system time on the server, tune ntp for reliability.  I also had this, and after a long time of diagnostics, such a huge error was found

**Comment by amal-viswa** [comment]
Hey, i'm having trouble while setting up azure ad authentication for wiki.js , I've followed the wiki.js official documentation for azure ad , but still not able to login error . can you guys help me resolve it

error- 

![Screenshot 2022-08-23 at 18-12-26 Error](https://user-images.githubusercontent.com/90927394/186365229-741e9d56-b488-400a-9518-a3939b2ef49d.png)

**Comment by LucasSPS** [comment]
https://github.com/requarks/wiki/discussions/4613#discussioncomment-14166586
